lib/frontend/app.js

```diff
@@ -126,12 +118,28 @@ module.exports = (appTokenGenerator) => {
       }
       Sentry.captureException(err)
```

Comment:
This needs `next(err)` so the error falls through to the next handler
```suggestion
       Sentry.captureException(err)
       next(err)
```